src/main/webapp/app/js/goal-page.js

```diff
@@ -69,9 +72,67 @@ function addComment() {
                 document.getElementById("userComment").value = "";
                 onLoad();
             } else {
-                alert("Sompthing went wrong! error:404");
+                alert("Something went wrong! error:404");
                 return false;
             }
         });
     }
+}
+
+function isGoalEditable() {
```

Comment:
Do you do this check on the backend as well? We can't rely on frontend for that purpose, because a clever user can modify it in the browser. Backend should perform its own validation of all incoming data.

Reply:
Yes, the backend also have validation for goal editing. If the user performs modifications in the browser and the goal he tries to edit is not his goal, then the goal is not edited.